packages/gatsby-plugin-netlify-cms/src/gatsby-browser.js

```diff
@@ -1,18 +1,33 @@
 /* global __PATH_PREFIX__ */
-import netlifyIdentityWidget from "netlify-identity-widget"
+
+// Taken from https://github.com/netlify/netlify-identity-widget
+const routes = /(confirmation|invite|recovery|email_change)_token=([^&]+)/
+const errorRoute = /error=access_denied&error_description=403/
+const accessTokenRoute = /access_token=/
```

Comment:
Instant bug if the source this was copied from changes. Perhaps Netlify Identity links should include a flag query like `netlify_identity_redirect=true`. Opened an issue to discuss: https://github.com/netlify/netlify-identity-widget/issues/157